workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/pages/AllScreens.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayLayout from '@clayui/layout';
+import {ClayVerticalNav} from '@clayui/nav';
+import {ReactNode} from 'react';
+
+import {useHash} from '../hooks/useHash';
+import {ScreenType} from '../types';
+import TicketsDashboard from './TicketsDashboard';
+import TicketsOverview from './TicketsOverview';
+
+const ROUTES = [
+	{
+		element: <TicketsDashboard screenType={ScreenType.INTEGRATED} />,
+		href: '#dashboard',
+		label: 'Dashboard',
+	},
+	{
+		element: <TicketsOverview />,
+		href: '#overview',
+		label: 'Tickets Overview',
+	},
+];
+
+const HREF_COMPONENT_MAP: {[key: string]: ReactNode} = {};
+
+ROUTES.forEach((route) => {
+	HREF_COMPONENT_MAP[route.href] = route.element;
+});
+
+const defaultRoute: string = '#dashboard';
```

Comment:
since this is used only once, there is no need to declare this const. It should be passed directly to useHash.